### PR TITLE
Empress Zoey writeup

### PR DESCRIPTION
**Pull request by amyrlam**
Things look wonky running locally with Docker, seems like Docker doesn't respect Markdown anymore(?). But I can verify the look of the Zoey on the Netlify after merge.

image

For your ref @mansona that we included this, put the link in again as it was broken in one spot last issue.

### CODE REVIEW

File: source/2019-08-16-the-ember-times-issue-111.md [hunk 1]
```diff
@@ -32,13 +38,16 @@ responsive: true
...
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-  
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+<div class="blog-row">
+  <img class="float-right small transparent padded" alt="Empress Zoey" title="Empress Zoey" src="/images/tomsters/empress.png" />
+
+You might recall that we wrote about [@mansona](https://github.com/mansona)'s talk about [Empress](https://www.youtube.com/watch?v=WL66D0bR0RQ&feature=youtu.be&t=780), Ember's static site/JAMStack product, last week. Hopefully you'll be inspired to build your own empress-blog template even if you only know HTML and CSS. **But wait, there's more!**
```
Comment (chrisrng) [inline source/2019-08-16-the-ember-times-issue-111.md]:
```suggestion
You might recall that we wrote about [@mansona](https://github.com/mansona)'s talk on [Empress](https://www.youtube.com/watch?v=WL66D0bR0RQ&feature=youtu.be&t=780), Ember's static site/JAMStack product, last week. Hopefully you'll be inspired to build your own [empress-blog](https://github.com/empress/empress-blog) template even if you only know HTML and CSS. **But wait, there's more!** 
```

**Review by chrisrng — APPROVED** [review]
Small proposed change otherwise looks good to me!

File: source/2019-08-16-the-ember-times-issue-111.md (hunk 1 above)
Comment (chrisrng) [inline source/2019-08-16-the-ember-times-issue-111.md]:
```suggestion
You might recall that we wrote about [@mansona](https://github.com/mansona)'s talk about [Empress](https://www.youtube.com/watch?v=WL66D0bR0RQ&feature=youtu.be&t=780), Ember's static site/JAMStack product, [last week](https://the-emberjs-times.ongoodbits.com/2019/08/09/issue-110). Hopefully you'll be inspired to build your own empress-blog template even if you only know HTML and CSS. **But wait, there's more!** 
```

Comment (chrisrng) [reply source/2019-08-16-the-ember-times-issue-111.md]: Should we add this? ^

Comment (amyrlam) [reply source/2019-08-16-the-ember-times-issue-111.md]: couldn't commit suggestion after accepting first one, so did manually

thanks, great edits!

**Comment by amyrlam** [comment]
The docker preview was correct, it has to be in HTML (missed that closing `<div>`, d'oh)! Fixed it on the main branch

**Comment by amyrlam** [comment]
image

fixed, putting date on main branch back to friday (blog doesn't appear cause date is in the future)

**Pull request merged**